core/domain/suggestion_services.py

```diff
@@ -1826,3 +2020,866 @@ def update_question_suggestion(
+            stat.submitted_questions_count)
+        stats_model.accepted_questions_count = (
+            stat.accepted_questions_count)
+        stats_model.accepted_questions_without_reviewer_edits_count = (
+            stat.accepted_questions_without_reviewer_edits_count)
+        stats_model.first_contribution_date = stat.first_contribution_date
+        stats_model.last_contribution_date = stat.last_contribution_date
+        stats_models_to_update.append(stats_model)
+
+    suggestion_models.QuestionContributionStatsModel.update_timestamps_multi(
+        stats_models_to_update,
+        update_last_updated_time=True)
+    suggestion_models.QuestionContributionStatsModel.put_multi(
+        stats_models_to_update)
+
+
+def _update_question_review_stats_models(
+    question_review_stats: List[
+        suggestion_registry.QuestionReviewStats
+    ]
+) -> None:
+    """Updates QuestionReviewStatsModel models for given question
+    review stats.
+
+    Args:
+        question_review_stats: list(QuestionReviewStats). A list of
+            QuestionReviewStats domain objects.
+    """
+    stats_dict = {}
+    for stat in question_review_stats:
+        stat_id = suggestion_models.QuestionReviewStatsModel.construct_id(
+            stat.contributor_user_id, stat.topic_id)
+        stats_dict[stat_id] = stat
+
+    stats_ids = stats_dict.keys()
+
+    stats_models = get_question_review_stats_models(list(stats_ids))
+    stats_models_to_update: List[
+        suggestion_models.QuestionReviewStatsModel] = []
+    for stats_model in stats_models:
+        # We can confirm that stats_model will not be None since we are
+        # checking None case in the get_question_review_stats_models
+        # function.
+        assert stats_model is not None
+        stat = stats_dict[stats_model.id]
+        stats_model.reviewed_questions_count = (
+            stat.reviewed_questions_count)
+        stats_model.accepted_questions_count = (
+            stat.accepted_questions_count)
+        stats_model.accepted_questions_with_reviewer_edits_count = (
+            stat.accepted_questions_with_reviewer_edits_count)
+        stats_model.first_contribution_date = stat.first_contribution_date
+        stats_model.last_contribution_date = stat.last_contribution_date
+        stats_models_to_update.append(stats_model)
+
+    suggestion_models.QuestionReviewStatsModel.update_timestamps_multi(
+        stats_models_to_update,
+        update_last_updated_time=True)
+    suggestion_models.QuestionReviewStatsModel.put_multi(
+        stats_models_to_update)
+
+
+def _get_date_as_string(date: datetime.datetime) -> datetime.date:
+    """Gets the string value of the given date.
+
+    Args:
+        date: datetime.datetime. Date that needs to converted into a string.
+
+    Returns:
+        datetime.date. The formatted object of the given date.
+    """
+    return datetime.datetime.strptime(str(
+        date.utcnow().date().isoformat()), '%Y-%m-%d'
+    ).date()
```

Comment:
Wait, looking at this again, why we need to do this bunch of stuff just to get datetime object again?

Reply:
Okay, I tried this code and it always seems to return today's date. Can you please fix this?

Reply:
> Wait, looking at this again, why we need to do this bunch of stuff just to get datetime object again?

I wanted to get the date in the format of yyyy-MM-dd.

> Okay, I tried this code and it always seems to return today's date. Can you please fix this?

I removed `utcnow()` and suggestion.last_updated always saved in the UTC time.